openslides/motions/static/js/motions/site.js

```diff
@@ -906,9 +907,21 @@ angular.module('OpenSlidesApp.motions.site', [
                 image_sources = image_sources.concat(tmp_image_sources);
             });
 
-            //post-request to convert the images. Async.
-            $http.post('/core/encode_media/', JSON.stringify(image_sources)).then(function (success) {
-                var converter = PdfMakeConverter.createInstance(success.data.images);
+            var image_map = [];
+            _.forEach(image_sources, function (image_source) {
```

Comment:
copy from below:

> Key is a string (the URL to an image) I is an integer.
> 
> Promise.all resolves our list of promises (to list of base64Str)
> with base64Str[i] we get the base64Str of the correct URL.

I need the following structure in some sort of map:

```
{
   URL_AS_STRING : RepresentativeBase64;
}
```

your suggestion would just bring
```
{
 0: PROMISE
}
```

I could then solve all thouse promises, would have lost the correct order of the URLs.

Is there a way to return "KEY : VALUE" in underscore JS?

roughly like this? 
```
var image_promises = _.map(image_sources, function (image_source) {
    return {source: PDFLayout.imageURLtoBase64(image_source)};
});
```
(code above is wrong)